source/WearableSensingDSI24.rst

```diff
@@ -0,0 +1,82 @@
+
+
+
+==================================================
+
+PhysioLab\ :sup:`XR` + Wearable Sensing's DSI-24
+
+==================================================
+
+This tutorial will demonstrate how to connect the DSI-24 to your computer and PhysioLab\ :sup:`XR`.
```

Comment:
please add a hyperlink to the DSI-24 website when it is mentioned in the docs; also, add a link for Wearable Sensing.